src/main.py

```diff
@@ -47,13 +50,14 @@ def main():
                 if left_points[i][2] < zone and left_points[i][0] % 4 == 0:
                     fingers.append((left_points[i], left_points[i-1]))
         if right_points:
-            for i in range(len(left_points)):
+            for i in range(len(right_points)):
                 if right_points[i][2] < zone and right_points[i][0] % 4 == 0:
                     fingers.append((right_points[i], right_points[i-1]))
 
         if fingers:
             for finger in fingers:
-                key_hash = finger[0][1]//hashs
+                key_hash = (finger[0][1]-int(m_width/50) -

```

Comment:
don't use division of const values in the cycle, you need to create a new variable with this value